clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -243,6 +247,9 @@ public void poll(long timeout, long now, PollCondition pollCondition) {
             // trigger wakeups after checking for disconnects so that the callbacks will be ready
             // to be fired on the next call to poll()
             maybeTriggerWakeup();
+            
+            //Throw InterruptException if this thread is interrupted
```

Comment:
nit: can we follow the comment convention used in the rest of this method (i.e. lower case first word and space after the '/')? We're not super consistent globally, but it would be nice to at least be consistent locally.